.github/workflows/quick-check.yml

```diff
@@ -6,6 +6,9 @@ on:
   push:
     branches-ignore:
       ["master", "auto", "try", "try-*", "dependabot/**"]
+  pull_request:
+    types: ['opened', 'synchronize']
+    branches: ["**"]
```

Comment:
I believe that for the merge queue to work properly the pull_request checks need to be the same as the ones that run for the merge queue (ie run all tests before merging). That's why the pull_request is now part of the main workflow.

Reply:
Having last step in separate workflow with same name seems to work in my test repo. Also results jobs from non-main workflows are useless as they were only used by bors to determine success or failure.

Reply:
But quick-check doesn't get run here so I reverted this and removing redundant result jobs should be followup PR.